### PR TITLE
test run component

### PR DESCRIPTION
**Pull request by evmiguel**
This renders the headings for the Test Run
Screen Shot 2020-04-28 at 5 54 50 PM
Screen Shot 2020-04-28 at 5 56 13 PM

### CODE REVIEW

File: server/services/UsersService.js [hunk 1]
```diff
@@ -213,7 +213,7 @@ async function getAllTesters() {
 }
```
Comment (evmiguel) [inline server/services/UsersService.js:213]: There was a bug in signup. It was not propagating the user id to the `req.session.user`

File: client/components/ManageCycles/index.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ class ManageCycles extends Component {
                     <title>Test Management | ARIA-AT</title>
                 </Helmet>
                 <h2>Initiate a Test Cycle</h2>
-                <Button as={Link} to="/new">
+                <Button as={Link} to="/cycles/new">
```
Comment (spectranaut) [inline client/components/ManageCycles/index.jsx:26]: whoops thanks for catching this!

File: client/components/TestQueue/index.jsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ class TestQueue extends Component {
         let designPatternLinkOrName;
         if (currentUserAssigned) {
             designPatternLinkOrName = (
-                <Link to={`/cycle/${cycleId}/run-tests/${run.id}`}>
+                <Link to={`/cycles/${cycleId}/run-tests/${run.id}`}>
```
Comment (spectranaut) [inline client/components/TestQueue/index.jsx:39]: we could change "run-tests" to "run" but only if you want to!

File: server/services/UsersService.js (hunk 1 above)
Comment (spectranaut) [reply server/services/UsersService.js:213]: ahh thanks for catching

**Pull request merged**